modules/@apostrophecms/schema/ui/apos/components/AposInputJoin.stories.js

```diff
@@ -0,0 +1,138 @@
+import {
+  withKnobs,
+  select,
+  optionsKnob as options,
+  boolean
+} from '@storybook/addon-knobs';
+
+import AposInputJoin from './AposInputJoin.vue';
+
+export default {
+  title: 'Inputs (Join)',
+  decorators: [ withKnobs ]
+};
+
+export const joinInput = () => {
+
+  const max = select(
+    'Limit', {
+      None: null,
+      One: 1,
+      Two: 2,
+      Three: 3
+    },
+    null
+  );
+
+  const prepoulate = select(
+    'Prepopulate', {
+      None: 0,
+      One: 1,
+      Two: 2,
+      Three: 3
+    },
+    null
+  );
+
+  const isDisabled = boolean('Is Disabled?', false);
+
+  return {
+    components: { AposInputJoin },
+    data () {
+      return {
+        field: {
+          name: 'myJoin',
+          label: 'Link to A Page',
+          help: 'Choose a page from the joiner to link to it',
+          type: 'join',
+          withType: 'product',
+          max
+        },
+        value: {
+          data: getData(prepoulate)
```

Comment:
I think this will need to be an array of `_id` values. Then in the component it hits an API to get the pieces. I assume that'll be the `getOne` route once for each _id. Later we'll need a way to send up multiple IDs.

Reply:
It won't need more than the title and the _id back from the request, though.

Reply:
Okay, so that's wrong. You'll have the full documents. Forget this part.